src/pages/tasks/TaskAssigneeSelectorModal.tsx

```diff
@@ -194,21 +194,24 @@ function TaskAssigneeSelectorModal({reports, task}: TaskAssigneeSelectorModalPro
                         undefined, // passing null as report because for editing task the report will be task details report page not the actual report where task was created
                         OptionsListUtils.isCurrentUser({...option, accountID: option?.accountID ?? -1, login: option?.login ?? ''}),
                     );
-
                     // Pass through the selected assignee
                     TaskActions.editTaskAssignee(report, session?.accountID ?? -1, option?.login ?? '', option?.accountID, assigneeChatReport);
                 }
-                Navigation.dismissModal(report.reportID);
+                InteractionManager.runAfterInteractions(() => {
+                    Navigation.dismissModal(report.reportID);
+                });
                 // If there's no report, we're creating a new task
             } else if (option.accountID) {
                 TaskActions.setAssigneeValue(
                     option?.login ?? '',
-                    option.accountID,
+                    option.accountID ?? -1,
```

Comment:
@allgandalf the accountID is ```number```. If it is number, I think we should fallback to ```-1``` as we did in other logics.

Reply:
I just refer to our contributor style docs:
https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#default-value-for-inexistent-ids

![image](https://github.com/user-attachments/assets/9cd2978c-a028-488a-8a4c-3ebb9bb363b7)

Reply:
I think that the fallback ```-1``` should be applied regardless the id is number or string since I just checked this [PR](https://github.com/Expensify/App/pull/42634), we are using ```-1``` as number, for example https://github.com/Expensify/App/pull/42634/files#diff-ea38863732cfd28f820d12c5b3f2633a8799cb797929c39af675b6a11452e9ee

Reply:
If ```-1``` is a valid account id, then we should fall back to ```0```, right?